examples/md-flexible/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ target_compile_definitions(
 )
 
 # --- display warnings if single-/multi-site mode is used with a functor type that has not been implemented ---
-if (MD_FLEXIBLE_MODE MATCHES "MULTI-SITE")
+if (MD_FLEXIBLE_MODE MATCHES "MULTISITE|multisite")
```

Comment:
tolower

Reply:
done in [53d6fad](https://github.com/AutoPas/AutoPas/pull/778/commits/53d6fad3ea476758572a183c8d9e849437fcdf9e)